### ISSUE TITLE
Building for Leonardo fails

### ISSUE THREAD

**Issue by saelo** [amperka/ino#44]
Hi,
I'm trying to compile and upload the blink example to a Arduino Leonardo but I'm getting stuck at the following error:

```
Searching for Board description file (boards.txt) ... /usr/share/arduino/hardware/arduino/boards.txt
Searching for Arduino lib version file (version.txt) ... /usr/share/arduino/lib/version.txt
Detecting Arduino software version ...  1.0 (1.0.1)
Searching for Arduino core library ... /usr/share/arduino/hardware/arduino/cores/arduino
Searching for Arduino standard libraries ... /usr/share/arduino/libraries
Searching for Arduino variants directory ... /usr/share/arduino/hardware/arduino/variants
Searching for avr-gcc ... /usr/local/sbin/avr-gcc
Searching for avr-g++ ... /usr/local/sbin/avr-g++
Searching for avr-ar ... /usr/bin/avr-ar
Searching for avr-objcopy ... /usr/bin/avr-objcopy
src/sketch.ino
Searching for Arduino lib version file (version.txt) ... /usr/share/arduino/lib/version.txt
Detecting Arduino software version ...  1.0 (1.0.1)
Scanning dependencies of src
Scanning dependencies of arduino
src/sketch.cpp
arduino/wiring_shift.c
arduino/wiring_pulse.c
arduino/WInterrupts.c
arduino/wiring.c
arduino/wiring_digital.c
arduino/wiring_analog.c
arduino/Tone.cpp
arduino/Stream.cpp
arduino/Print.cpp
arduino/WMath.cpp
arduino/new.cpp
arduino/USBCore.cpp
/usr/share/arduino/hardware/arduino/cores/arduino/USBCore.cpp:78: error: ‘USB_VID’ was not declared in this scope
/usr/share/arduino/hardware/arduino/cores/arduino/USBCore.cpp:78: error: ‘USB_PID’ was not declared in this scope
/usr/share/arduino/hardware/arduino/cores/arduino/USBCore.cpp:81: error: ‘USB_VID’ was not declared in this scope
/usr/share/arduino/hardware/arduino/cores/arduino/USBCore.cpp:81: error: ‘USB_PID’ was not declared in this scope
make: *** [.build/arduino/USBCore.o] Error 1
Make failed with code 2
```

It happends with both, the avr-gcc from the repos (Arch Linux) and the avr-gcc shipped with arduino...

**Comment by nkrkv** [comment]
Well, supporting Leonardo is yet to be done issue. Thanks for the report.

**Comment by divadrei** [comment]
@sciepy try this https://github.com/jboone/ino work for me

**Comment by nkrkv** [comment]
@oxys Can you confirm that it works for _uploading_ as well?

**Comment by divadrei** [comment]
@nailxx I have a fix for uploading, I will fork tomorrow

**Comment by divadrei** [comment]
@nailxx fix for build and upload https://github.com/fablab-pau/ino, send me issue @fablab-pau if you find a new bug

**Comment by nkrkv** [comment]
Fixed in master starting with 347bc0e601edb3bcc61f54b7065b712f35c65ba7
